protocol/avdl/chat1/notify.avdl

```diff
@@ -57,4 +57,12 @@ protocol NotifyChat {
   @notify("")
   @lint("ignore")
   void ChatIdentifyUpdate(keybase1.CanonicalTLFNameAndIDWithBreaks update);
+
+  @notify("")
+  @lint("ignore")
```

Comment:
The capitalization is all of this in this file, I just decided to keep it consistent in-file and ignore the linter.